src/main/backend.ts

```diff
@@ -326,9 +326,9 @@ export class MainBackend extends BaseBackend<ElectronOptions> implements CommonB
           scope.setExtras(sentScope._extra);
         }
 
-        // Since we do not have updates for individual breadcrumbs anymore and only for the whole scope
+        // Since we do not have updates for individual breadcrumbs any more and only for the whole scope
         // we just add the last added breadcrumb on scope updates
-        scope.addBreadcrumb(sentScope._breadcrumbs.pop());
+        scope.addBreadcrumb(sentScope._breadcrumbs.pop(), this._options.maxBreadcrumbs);
```

Comment:
@kamilogorek is it required to pass `maxBreadcrumbs` down like this? 

There are a few open issues (#277, #205) about excessive breadcrumbs and although I haven't seen this myself in a long time, it seems like this might be required?

Reply:
🤦‍♂️ Just seen you've already replied...

Reply:
Tbh this https://github.com/getsentry/sentry-javascript/blob/master/packages/hub/src/hub.ts#L45-L49 should be moved to `scope.ts`, and logic could be slightly modified in both places (`hub/scope`), which would at least prevent those overacumulated leaks.